### DISCUSSION TITLE
Arabic transition for the README!

### DISCUSSION THREAD

**Discussion by Abdeltwab** [pointfreeco/swift-composable-architecture#933] [Ideas]
There are several Translations for the Repo ReadMe, and we  , @NorhanBoghdadi and myself,  would like to contribute to the project by translating the repo ReadME to  Arabic, to help to spread the project between the Arabic speaker's developers community.

we already started doing this, but wanted to stop working on dark and let the project community know about the idea and ask about the visibility to do this? 

also, What is the right way to let other contributors know that, we are already working on this, to avoid any conflicts or duplication if there is some ongoing work with the same goal?

**Comment by mbrandonw** [comment]
Hi @Abdeltwab, we would happily accept an Arabic translation of the README. The best way to contribute this would be to put the translation in a public gist, and then open a PR to this repo adding it in the [translations](https://github.com/pointfreeco/swift-composable-architecture#translations) section.

**Reply by Abdeltwab** [reply]
thanks  @mbrandonw
